Yoast/javascript
# Add function words for Indonesian

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo N/A] Add Indonesian function words.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to content analysis
* Run yarn start
* Set locale to id_ID.
* Paste an Indonesian text into the example.
* Make sure you see no strange word combinations, counts, etc.

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-412
